keyboard.js

```diff
@@ -0,0 +1,68 @@
+class Keyboard {
+    constructor(keySet, language) {
+
+        this.isCapsLock = false;
+        this.isShift = false;
+        this.language = language;
+        this.keys = keySet;
+        this.controlKeys = [
+            'Backspace',
+            'Tab',
+            'CapsLock',
+            'Enter',
+            'ShiftLeft',
+            'ShiftRight',
+            'ControlLeft',
+            'ControlRight',
+            'ContextMenu',
+            'AltLeft',
+            'AltRight',
+        ];
+    }
+
+    keySet() {
+        return this.keys;
+    }
+
+
+    checkControlKey(code) { /////////////////////////////
+        return this.controlKeys.some((controlKey) => controlKey === code); // some проверяет на соответствие условия заданного в передаваемой функции
+
+    }
+
+    checkCapsLock() {
```

Comment:
since checkCapsLock and checkLanguage just return values, it would be better to name them getCapsLock, getLanguage